neard/Cargo.toml

```diff
@@ -14,13 +14,14 @@ path = "src/../src/main.rs"
 name = "near"
 
 [dependencies]
-clap = "2.32"
+clap = "=3.0.0-beta.2"
```

Comment:
Any specific reason for exact version dependency?

Reply:
I guess it is a good idea to pin pre-release versions to avoid breaking changes causing compilation errors. It is less relevant for binary crates, though. Still, I guess it is a good rule of thumb.

Reply:
Yeah, I'm just not confident that there will be no breaking changes coming from the beta release so my intuition is that this would be more safe. This isn't used as a lib so the pin shouldn't matter